### PR TITLE
Mmansurova/abc 262

### PR DESCRIPTION
**Pull request by Mil-m**

### CODE REVIEW

File: src/app/globalsearch/globalsearch.component.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Search</title>
+</head>
+<body>
```
Comment (Alexey-ebi) [inline src/app/globalsearch/globalsearch.component.html:6]: you don't need to provide this, only html code that is relevant to the component itself. These tags are already provided here - https://github.com/FAANG/dcc-portal-frontend/blob/master/src/index.html

File: src/app/shared/header/header.component.html [hunk 2]
```diff
@@ -156,3 +155,13 @@
     </a>
   </button>
 </mat-menu>
+
+<!-- Search Menu Dropdown -->
+<mat-menu #search="matMenu">
+  <button mat-menu-item>
+    <a class="nav-link" routerLink="/graphql" routerLinkActive="active">Search</a>
```
Comment (Alexey-ebi) [inline src/app/shared/header/header.component.html:162]: can we rename this to "Custom queries"

File: src/app/shared/header/header.component.html [hunk 3]
```diff
@@ -156,3 +155,13 @@
...
+  </button>
+  <button mat-menu-item>
+    <a class="nav-link" routerLink="/globalsearch" routerLinkActive="active">Global Search</a>
```
Comment (Alexey-ebi) [inline src/app/shared/header/header.component.html:165]: can we rename this to just Search

File: src/app/globalsearch/globalsearch.component.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      </td>
+      <td>
+        <p>Link:
+          <a [routerLink]="['/' + item.key ]" [queryParams]="{ searchTerm: searchText }">
+            {{ item.key + '?searchTerm=' + searchText }}
+          </a>
+        </p>
+      </td>
+    </tr>
+  </table>
```
Comment (Alexey-ebi) [inline src/app/globalsearch/globalsearch.component.html:36]: I don't we need a table here, can you convert this to a Angular material list please

File: src/app/globalsearch/globalsearch.component.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        <p>Index: {{ item.key }}</p>
+      </td>
+      <td>
+        <p>Total Hits: {{ item.value.totalHits }}</p>
+      </td>
+      <td>
+        <p>Link:
+          <a [routerLink]="['/' + item.key ]" [queryParams]="{ searchTerm: searchText }">
+            {{ item.key + '?searchTerm=' + searchText }}
+          </a>
```
Comment (Alexey-ebi) [inline src/app/globalsearch/globalsearch.component.html:32]: that should be <n> <index_name> found and <index_name> will be a link, ex.:

34 <a href="/organism/..."> organisms</a> found

**Review by Alexey-ebi — CHANGES_REQUESTED** [review]

File: src/app/globalsearch/globalsearch.component.html (hunk 1 above)
Comment (yroochun) [reply src/app/globalsearch/globalsearch.component.html:6]: We need to add the app-header component to this page to display FAANG toolbar

File: src/app/shared/header/header.component.html (hunk 3 above)
Comment (Mil-m) [reply src/app/shared/header/header.component.html:165]: we have just 'Search' for the graphql there

**Comment by yroochun** [comment]
1. When navigating to the global search, please update the title

2. When performing a search, update the url to have the search term. This will enable user to return to his previous search when navigating to the results page

**Pull request merged**